api/Element.json

```diff
@@ -1956,7 +1956,7 @@
               "version_added": true
             },
             "chrome_android": {
-              "version_added": false
+              "version_added": true
```

Comment:
This also applies to WebView.

Reply:
Here's a list of what's [not exposed in WebView](https://cs.chromium.org/chromium/src/android_webview/tools/system_webview_shell/test/data/webexposed/not-webview-exposed.txt).

Reply:
@jpmedley Sorry, I had a look but I don't know what to do with this. Do you have an example of how to mark up both separately?

Reply:
Down below, there’s an entry for “webview_android”.  If you can set that to “true” as well, that would be great!

Reply:
D'oh, of course.